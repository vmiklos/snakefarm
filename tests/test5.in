debug 1
out 1
time 100
load test5.map
addplayer 1
addsnake 1 1 0 1 1 0
16 0
15 0
14 0
endsnake
addplayer 2
addsnake 2 2 1 1 1 1
9 0
endsnake
step
comment # az elozo lepesben az elso kigyo 3. egysege leszakadt, a 2. megevodott, a 3.-bol uj kigyo keletkezett 3 sorszammal
showplayer 1
exit