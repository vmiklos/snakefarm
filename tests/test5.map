1;W;0;0
2;W;0;1
3;W;0;2
4;W;0;3
5;W;0;4
6;W;0;5

7;W;1;0
8;0;2;7
9;0;3;8
10;0;4;9
11;0;5;10
12;W;6;11

13;W;7;0
14;0;8;13
15;0;9;14
16;0;10;15
17;0;11;16
18;W;12;17

19;W;13;0
20;W;14;19
21;W;15;20
22;W;16;21
23;W;17;22
24;W;18;23

