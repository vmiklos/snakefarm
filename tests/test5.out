Event MapLoad test5.map
Event Step 1
End Step
Event Step 2
StepEvent Snake_Snake 1
StepEvent SnakeSplit 2 3
End Step
# az elozo lepesben az elso kigyo 3. egysege leszakadt, a 2. megevodott, a 3.-bol uj kigyo keletkezett 3 sorszammal
Player 1
1 1 0 1 1
3 1 0 1 1
endplayer
Event Exit